scripts/setup-wsl2.ps1

```
@@ -0,0 +1,50 @@
+# =============================================================================
+# setup-wsl2.ps1 — Install OpenROAD-MCP dependencies on Windows via WSL2
+# =============================================================================
+
+Write-Host "🔧 Setting up OpenROAD-MCP on Windows via WSL2..." -ForegroundColor Cyan
+
+# Check WSL2 is available
+$wslVersion = wsl --status 2>$null
+if ($LASTEXITCODE -ne 0) {
+    Write-Host "❌ WSL2 is not installed. Run: wsl --install" -ForegroundColor Red
```

PSScriptAnalyzer (1.24.0)

[warning] File 'setup-wsl2.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)